File: P/app/src/main/java/com/project/sns/ui/adapters/listener/onClickItemListener.java
package com.project.sns.ui.adapters.listener;

import com.project.sns.data.write.PostData;

import java.util.List;

public interface onClickItemListener {
    public void onClickItem(int position, List<PostData> postData);
}
interface onClickItemAdapter {
    public void onClickAdapter(int position, List<PostData> postData);
}

File: P/app/src/main/java/com/project/sns/ui/adapters/listener/onClickItemListener.kt
package com.project.sns.ui.adapters.listener;

import com.google.firebase.storage.StorageReference
import com.project.sns.data.board.PostData;

 interface onClickItemListener {
    fun onClickItem(position : Int,  postData : ArrayList<PostData>);
}

interface onClickImageListener {
    fun onClickImage(data: StorageReference)
}

interface onClickItemAdapter {
    fun onClickAdapter(position : Int,  postData : ArrayList<PostData>);
}
